experiments/ext_api/word_count_demo/word_count_app.py

```diff
@@ -60,7 +60,6 @@ def count_word(word, word_totals):
 
 class WordTotal(object):
     def __init__(self):
-        print("word total")
```

Comment:
@strmpnk was this `print` intentional?

Reply:
I've removed this locally but yes, it was intentional logging for awhile (there was a very odd bug I had to track down during the initial run awhile back so I wanted to ensure that was getting constructed).